tests/console/yast2_http.pm

```diff
@@ -102,25 +103,17 @@ sub run {
     assert_screen 'http_vitual_host_page';          # check wizard page (4/5)
     send_key 'alt-n';                               # go to http server wizard (5/5) --summary
                                                     # make sure that apache2 server got started when booting
-    if ((is_sle '<15') || (is_leap)) {
+    if (is_sle('<15') || is_leap('<15.1')) {
         assert_screen 'http_summary';
         send_key 'alt-t';
         assert_screen 'http_start_apache2';
     }
     else {
-        assert_screen 'yast2_ncurses_service_start_widget';
-        send_key 'alt-t';
-        send_key_until_needlematch 'yast2_ncurses_service_start_widget_start_after_conf', 'up';
-        send_key 'ret';
-        assert_screen 'yast2_ncurses_service_start_widget_check_start_after_conf';
-        send_key 'alt-a';
-        send_key_until_needlematch 'yast2_ncurses_service_start_widget_start_on_boot', 'up';
-        send_key 'ret';
-        assert_screen 'yast2_ncurses_service_start_widget_check_start_on_boot';
+        $self->SUPER::assert_service_widget(after_writing_conf => 'alt-w', after_reboot => 'alt-e');
```

Comment:
I don't think we need `SUPER` here, as you don't override method in inherited class.